<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDlg</class>
 <widget class="QDialog" name="MainDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>290</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>570</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>570</width>
    <height>290</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Picture Message</string>
  </property>
  <widget class="QPlainTextEdit" name="textEditBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>251</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="loadImageBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="insertTextBtn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Insert text to image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="extractTextBtn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Extract text from Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>92</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveImageBtn">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Image</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>131</y>
     <width>111</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Filename:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Image Size :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Size needed:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Size Free:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>414</x>
     <y>131</y>
     <width>141</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <item>
     <widget class="QLabel" name="fileNameLable">
      <property name="text">
       <string>--</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="imageSizeLabel">
      <property name="text">
       <string>--</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="sizeNeededLabel">
      <property name="text">
       <string>--</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="sizeFreeLabel">
      <property name="text">
       <string>--</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
